ichnaea/db.py

```diff
@@ -19,29 +21,35 @@
 from sqlalchemy.sql import func, select
 from sqlalchemy.sql.expression import Insert
 
-from ichnaea.config import (
-    ALEMBIC_CFG,
-    DB_LIBRARY,
-    DB_DDL_URI,
-    DB_RW_URI,
-    DB_RO_URI,
-)
+from ichnaea.conf import settings
+
 
 DB_TYPE = {
-    None: DB_DDL_URI,
-    'ddl': DB_DDL_URI,
-    'ro': DB_RO_URI,
-    'rw': DB_RW_URI,
+    # Default to readonly if db_type isn't specified
+    None: settings('db_readonly_uri'),
```

Comment:
The default used to be the ddl uri, but that seems like a really bad idea to me, so I changed it to the readonly uri.

Reply:
My guess would be a failure in a setup or DB migration script rather than app code, which may include IT-managed scripts that aren't in this repo.

I think the chance of this breaking something is a lot less in the dockerized environments, and I like that it breaks safely rather than unsafely.